Gemfile

```diff
@@ -26,7 +26,6 @@ gem 'apitome'
 gem 'dry-container'
 gem 'dry-auto_inject'
 gem 'dry-validation'
-gem 'thin'
```

Comment:
thin is still ok for development and testing

Reply:
We can just leave it, but I thought we use WEBrick in these envs.

Reply:
ah, yes
okay, let it be